### ISSUE TITLE
Allow Bluetooth access

### ISSUE THREAD

**Issue by tinywrkb** [flathub/com.google.Chrome#18]
I'm trying to get `Nearby Sharing` working without much success. On one occasion `chrome://nearby` somehow listed some users but not my own device.

Bluetooth is a precondition for testing this feature so it might be a good idea to enable this by adding:

```
- --allow=bluetooth
- --system-talk-name=org.bluez
```

I tested Chrome with the `Web Bluetooth API` sample [here](https://googlechrome.github.io/samples/web-bluetooth/device-info.html?allDevices=true) which needs the `Experimental Web Platform features` Chrome flag enable.  
The linked sample test code actually only needs the `system-talk-name` permission and doesn't work with the `allow` permission, not sure what's going on there.

**Comment by refi64** [comment]
Do you actually need `--allow=bluetooth`? That should just give access to AF_BLUETOOTH sockets which I don't *think* are needed, at least from a quick check of the code it probably just talks to bluez for everything.

**Comment by tinywrkb** [comment]
Right, it doesn't look like `--allow=bluetooth` is needed. Note that it's possible to check the Bluetooth status via `chrome://bluetooth-internals`.  
It seems like `Nearby Sharing` has been removed from the unstable builds, so this and the fact `Web Bluetooth API` is still experimental makes me question this request, maybe close this issue for the time being.

**Comment by Danik1601** [comment]
Developers [can use](https://support.google.com/chrome/answer/6362090) Bluetooth on the web sites and [PWA](https://en.wikipedia.org/wiki/Progressive_web_app)s now


Google itself now [actively uses](https://9to5google.com/2023/05/03/google-account-passkeys/#:~:text=If%20you%E2%80%99re%20signing%20in%20with%20a%20new%20device%20(or%20doing%20so%20temporarily)%2C%20you%20use%20the%20passkey%20on%20your%20phone%20with%20a%20QR%20code%20scanning%20process%20and%20a%20Bluetooth%20proximity%20check.) Bluetooth for things like fast sign-in using Android devices
